autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -4,18 +4,89 @@
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
+from course import generate_course_id
 
-def parse_section(section):
-    " parse_course() must be done first, and pass the data for each section here "
+def parse_section(course):
+    #gets values to create object
+    section_id = ''
+    section_subject = course['subject']
+    course_number = course['courseNumber']
+    section_number = ''
+    section_term_code = course['term']
+    
+    section_min_credits = course['creditHourLow']
+    section_max_credits = course['creditHourHigh']
 
-def parse_meeting(meeting):
-    " parse_section() must be done first, and pass the data for each meeting here"
+    section_max_enrollment = course['maximumEnrollment']
+    section_current_enrollment = course['enrollment']
+    section_instructor = ''
```

Comment:
Unsure of what to do in this case because faculty is a list and this is a string field. In some courses, there might be more than one faculty member listed. For now I just made it so it puts all the instructor names into a list and outputs that as a string. Also, I just realized this results in an empty string sometimes. I'ma fix it real quick.

Reply:
Fixed the empty string bug. Is there a way to add that to this pull request or do I have to make a new one? If I have to make a new one, is it possible to keep these comments or do I have to copy paste them to the new request?

Reply:
Just add a commit to the same branch and the PR will update automatically